src/app/containers/VestedHistory/index.tsx

```diff
@@ -132,31 +132,31 @@ export function VestedHistory() {
   ]);
 
   return (
-    <div className="sovryn-table tw-p-4 tw-mb-12">
+    <div className="sovryn-table">
       <table className="tw-w-full">
         <thead>
           <tr>
-            <th className="tw-text-left assets">
+            <th className="tw-text-left tw-font-light assets">
               {t(translations.vestedHistory.tableHeaders.time)}
             </th>
-            <th className="tw-text-left">
+            <th className="tw-text-left tw-font-light">
               {t(translations.vestedHistory.tableHeaders.vestingSchedule)}
```

Comment:
![UI issue1](https://user-images.githubusercontent.com/56412037/144451736-b51479cf-7a47-42b8-910f-78ff61d356ce.JPG)
In UI/UX, it's title is "Asset", but now it is saying "Type"